docs/administration/configuration/configuring-access/index.md

```diff
@@ -24,8 +24,64 @@ Helm value accordingly.
 
 ## User registration via Single Sign-On
 
-Handling of users with Single Sign-On (SSO) is also possible. Currently,
-this is only available for CERN deployments via [`components.reana_ui.cern_sso`](https://github.com/reanahub/reana/tree/master/helm/reana)
+Handling of users with Single Sign-On (SSO) is also possible. Currently SSO is supported for CERN as well as for any third-party SSO provider which supports [Keycloak](https://www.keycloak.org/).
+
+### Keycloak Single Sign-On Configuration
+
+First of all, to integrate REANA with your Keycloak instance, you need to create a new client from the Keycloak's admin dashboard.
+There are many configuration options, but these are the minimal changes you have to make:
+
+- _Valid Redirect URIs_ should be set to `https://<your-reana-hostname>/api/oauth/authorized/keycloak/` (see Keycloak's [Access Settings](https://www.keycloak.org/docs/latest/server_admin/#access-settings))
+- _Client authentication_ should be enabled in order to get the client ID and the client secret (see Keycloak's [Confidential client credentials](https://www.keycloak.org/docs/latest/server_admin/#_client-credentials))
+
+You can then configure REANA to use your third-party SSO provider by adding an item with the needed configuration to the [`login`](https://github.com/reanahub/reana/tree/master/helm/reana) list in the Helm values:
+
+- `name` can be chosen freely, and it will be used as an identifier for the SSO provider
+- `type` must be set to `keycloak`
+- `config.title` is the name of the SSO provider that will be shown to users on the web interface, for example in the login button
+- `config.base_url`, `config.realm_url`, `config.auth_url`, `config.token_url` and `config.userinfo_url` should all be set to the corresponding Keycloak endpoints
+
+Please note that currently only one provider of `keycloak` type is supported, so the `login` array must contain at most one element:
+
+```{ .yaml .copy-to-clipboard }
+login:
+  - name: "your-provider"
+    type: "keycloak"
+    config:
+      title: "YOUR PROVIDER"
+      base_url: "https:/<your-provider-hostname>"
+      realm_url: "https://<your-provider-hostname>/auth/realms/your-realm"
+      auth_url: "https://<your-provider-hostname>/auth/realms/your-realm/protocol/openid-connect/auth"
+      token_url: "https://<your-provider-hostname>/auth/realms/your-realm/protocol/openid-connect/token"
+      userinfo_url: "https://<your-provider-hostname>/auth/realms/your-realm/protocol/openid-connect/userinfo"
+```
+
+Then specify the provided client ID and client secret under the [`secrets.login`](https://github.com/reanahub/reana/tree/master/helm/reana) Helm value:
+
+```yaml
+secrets:
+  login:
+    your-provider:
+      consumer_key: <your-client-id>
+      consumer_secret: <your-client-secret>
+```
+
+!!! note
```

Comment:
The note could be turned into a regular text:

Note that the key "your-provider" must match the internal identifier name you have chosen above as the name of your SSO provider.